source/client/controllers/login-controller.js

```diff
@@ -8,7 +8,8 @@ Space.messaging.Controller.extend(Space.accountsUi, 'LoginController', {
   eventSubscriptions() {
     return [{
       'Space.accountsUi.LoginRequested': this._onLoginRequested,
-      'Space.accountsUi.LogoutRequested': this._onLogoutRequested
+      'Space.accountsUi.LogoutRequested': this._onLogoutRequested,
+      'Space.accountsUi.LoginWithGoogleRequested': this._onLoginWithGoogleRequested,
```

Comment:
But there should be one LoginWithServiceRequested event for all services or the solution with one event as you are suggesting.